packages/design-system/src/components/Form/Field/Input/Input.Checkbox.tsx

```diff
@@ -100,6 +100,7 @@ export const SCheckbox = styled(InlineStyle)<{
 
 export type CheckboxProps = InputProps & {
 	checked?: boolean | 'indeterminate' | (string | number)[];
+	'data-feature'?: string;
```

Comment:
We should be able to pass any data attribute we want, it's a native HTML element we are trying to imitate here. `data-feature` is supposed to be spread within `...rest`.
A better approach would be to change these proptypes to reflect that.

Reply:
that's the point of this change: the data-feature has a custom behavior, it's not forwared to the label + it's suffixed with a state specific value

Reply:
Alright, I'm stil uncomfortable with it. Why would a random data-attribute duplicate what is essentially the state of that component and what could essentially be its value? 
Are we OK with shared components having such highly specific behaviours hidden behind data-attributes?